drivers/unifiedPlatform/stworkflows/workflowPlatformRBAC.go

```diff
@@ -1,67 +0,0 @@
-package stworkflows
```

Comment:
Please don't delete this file, this is required for platform RBAC

Reply:
Added back